apps/cowswap-frontend/src/common/updaters/orders/SpotPricesUpdater.ts

```diff
@@ -87,7 +87,9 @@ export function SpotPricesUpdater(): null {
         return
       }
 
-      const fraction = inputPrice.price.asFraction.divide(outputPrice.price.asFraction)
+      const inputFraction = FractionUtils.fromPrice(inputPrice.price)
+      const outputFraction = FractionUtils.fromPrice(outputPrice.price)
```

Comment:
Now using the correct `fromPrice` method, which takes care of the decimals conversion properly.

Reply:
Prove it :) Add a unit test